mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -278,6 +278,11 @@ SONIC:
 		Offset: -170,0,0
 	AutoTarget:
 	WithVoxelTurret:
+	Explodes:
+		RequiresCondition: !rank-elite
+	Explodes@ELITE:
+		RequiresCondition: rank-elite
+		Weapon: UnitExplode
```

Comment:
I meant not why you did it, but why/how it is done, that once it is inherited, once default value used, I must miss something.